server/code/setup/cpdb

```diff
@@ -131,6 +131,18 @@ class PostgresqlSetup(DbSetup):
             # Append / for the database name:
             self.jdbc_url = "%s/" % (self.jdbc_url)
         self.jdbc_url = "%s%s" % (self.jdbc_url, db)
+
+        # Appending SSL connection details
+        if ssl == True and sslfactory is None and sslcertpath is None:
+            self.jdbc_url = "%s%s" % (self.jdbc_url, "?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory")
+            print("Default sslfactory=org.postgresql.ssl.NonValidatingFactory is used as you have not provided factory details.")
+
+        elif ssl == True and sslfactory is not None and sslcertpath is None:
+            self.jdbc_url = "%s%s%s" % (self.jdbc_url, "?ssl=true&sslfactory=",sslfactory)
```

Comment:
Missing space between last two arguments in the format tuple.

Reply:
Updated as suggested.